### PR TITLE
Modifying filter_elements in HeapedVector.jl

### PR DESCRIPTION
**Pull request by yashcodes**

### CODE REVIEW

**Comment by dpsanders** [comment]
Please checkout and pull the master branch and do
```
git rebase master
```
from this branch to get rid of the previous commits.

**Comment by dpsanders** [comment]
Or just checkout latest master, make a new branch and use `git cherry-pick`. 
Or just make the change again!

File: src/HeapedVectors.jl [hunk 1]
```diff
@@ -91,17 +91,8 @@ function bubbledown!(v::HeapedVector{T}, index) where{T}
...
-    filter!(func, A.data)
-
-    if length(A.data) == 0
-        return A
-    end
-
-    heaping(A.data, A.by)
-    return A
-
+function filter_elements!(v::SortedVector{T}, x::T) where {T}
```
Comment (dpsanders) [inline src/HeapedVectors.jl]: This should be `HeapedVector`?

Comment (yashcodes) [reply src/HeapedVectors.jl]: Yes it should be.!!

**Pull request closed without merging**